introduction_to_applying_machine_learning/xgboost_customer_churn/xgboost_customer_churn.ipynb

```diff
@@ -49,7 +49,6 @@
    "cell_type": "code",
```

Comment:
<p>add an import for sagemaker.inputs.TrainingInput</p><p><code>from sagemaker.inputs import TrainingInput</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>